pom.xml

```diff
@@ -121,6 +122,18 @@
       <scope>test</scope>
       <version>${reactor-test.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>elasticsearch</artifactId>
+      <version>${org.testcontainers.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>junit-jupiter</artifactId>
```

Comment:
@hizmailovich what is for

Reply:
To start elasticsearch container, I don't remember exactly, for what annotation

Reply:
@hizmailovich you can start without this dependency